test/hotspot/jtreg/serviceability/dcmd/vm/SystemMapTest.java

```diff
@@ -28,7 +28,21 @@
 import jdk.test.lib.process.OutputAnalyzer;
 
 /*
- * @test
+ * @test id=zgc
+ * @bug 8346717
+ * @requires vm.gc.Z
+ * @summary Test of diagnostic command System.map using ZGC
+ * @library /test/lib
+ * @requires (os.family == "linux" | os.family == "windows" | os.family == "mac")
+ * @modules java.base/jdk.internal.misc
+ *          java.compiler
+ *          java.management
+ *          jdk.internal.jvmstat/sun.jvmstat.monitor
+ * @run testng/othervm -XX:+UsePerfData -XX:+UseZGC SystemMapTest
+ */
+
+/*
+ * @test id=normal
```

Comment:
The normal test should be for non-ZGC otherwise in a test config that runs everything with ZGC you will run them both when only one is needed